### PR TITLE
UNI-153 #comment Migrate real Metric schema from Taurus/Grok, test... WIP

### PR DESCRIPTION
**Pull request by brev**
UNI-153 #comment Migrate real Metric schema from Taurus/Grok, test real
validation. Put in sublevels for DB (akin to database tables), and put in
json query engine for winnowing queries. Full examples inside. 
Will continue importing rest of Taurus/Grok DB schema next.

FYI @lscheinkman @marionleborgne @vitaly-krugl

### CODE REVIEW

**Comment by brev** [comment]
WIP

File: unicorn/frontend/browser/app.js [hunk 1]
```diff
@@ -65,29 +66,28 @@ let context;
...
+  "datasource": "blah",
+  "name": "blah",
+  "description": "blah",
+  "server": "blah",
+  "location": "blah",
+  "parameters": "blah",
+  "status": 0,
+  "message": "blah",
+  "collector_error": "blah",
+  "last_timestamp": "2015-09-03T23:56:16+00:00",
```
Comment (vitaly-krugl) [inline unicorn/frontend/browser/app.js]: @brev, you shouldn't take database fields from grok/taurus/htmengine verbatim. There are many here that are relics of Grok and other baggage that don't apply to unicorn at all. I recommend starting with what you need immediately and adding what you need as the project progresses.

Comment (vitaly-krugl) [inline unicorn/frontend/browser/app.js]: For example, the following are largely holdovers from Grok's AWS/CloudWatch-specific functionality:
* last_timestamp
* server
* location
* collector_error
* poll_interval
* tag_name

**Comment by brev** [comment]
@lscheinkman @marionleborgne please review anytime.  The DatabaseClient/Server methods should now be working if you want to start persisting your CSV files. thanks!

**Pull request merged**